--- Metadata-0.23/examples/gather~	Mon Jul 27 15:57:16 1998
+++ Metadata-0.23/examples/gather	Sat Oct 16 19:40:00 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # $Source$
 #
--- Metadata-0.23/examples/htdig-dump~	Mon Jul 27 15:51:53 1998
+++ Metadata-0.23/examples/htdig-dump	Sat Oct 16 19:40:14 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # $Source$
 #
--- Metadata-0.23/examples/scan-log~	Mon Jul 27 15:50:31 1998
+++ Metadata-0.23/examples/scan-log	Sat Oct 16 19:40:23 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # $Source$
 #
